jiant/preprocess.py

```diff
@@ -282,9 +284,9 @@ def build_tasks(args):
         setattr(task, "_classifier_name", task_classifier if task_classifier else task.name)
 
     tokenizer_names = {task.name: task.tokenizer_name for task in tasks}
-    assert len(set(tokenizer_names.values())) == 1, (
-        f"Error: mixing tasks with different tokenizers!" " Tokenizations: {tokenizer_names:s}"
-    )
```

Comment:
Out of curiosity, how could this ever be triggered? If we're allowing people to set task += {tokenizer_name ...}, we should just stop allowing that.